python/lsst/ts/MTAirCompressor/simulator.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of ts_MTAirCompressor.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["create_server"]
+
+from pymodbus.server.sync import ModbusTcpServer
+from pymodbus.datastore import (
+    ModbusSequentialDataBlock,
+    ModbusSlaveContext,
+    ModbusServerContext,
+)
+
+
+def create_server():
+    """Create simulator server.
+
+    Returns
+    -------
+    server : `ModbusTcpServer`
+        Created server instance.
+    """
+    store = ModbusSlaveContext(hr=ModbusSequentialDataBlock(0, [47] * 0xA000))
+    context = ModbusServerContext(slaves=store, single=True)
+
+    return ModbusTcpServer(context, address=("localhost", 5020))
```

Comment:
Do you want to consider to put the hostname and port into the arguments of `create_server()`?

Reply:
I consider that but reject it for the moment. I don't need it now, and don't think we will need it in the future - if we will (particularly for port number; hostname shall be either localhost or FQDN anyway, anything else will not work), it's easy enough to add that.